toucan_connectors/revinate/revinate_connector.py

```diff
@@ -0,0 +1,24 @@
+"""
+Revinate connector
+
+Documentation can be found at: https://porter.revinate.com/documentation
+
+This connector inherits from the HTTP connector classes rather than the base toucan-connector classes
+"""
+import pandas as pd
+
+from toucan_connectors.http_api.http_api_connector import HttpAPIConnector, HttpAPIDataSource
+
+
+class RevinateDataSource(HttpAPIDataSource):
+    base_url = 'https://porter.revinate.com'
```

Comment:
I think that should go into the `RevinateConnector` and not the `RevinateDataSource`
(because `baseroute` is a property of `HttpApiConnector` and not `HttpApiDataSource`, see https://github.com/ToucanToco/toucan-connectors/blob/master/toucan_connectors/http_api/http_api_connector.py#L87)